src/common/Widget.js

```diff
@@ -578,6 +579,29 @@
     };
 
     //  DOM/SVG Node Helpers  ---
+    
+    Widget.prototype.showBusy = function (show) {
+        if (show) {
+            //show the indicator
+        } else {
+            //hide the indicator
```

Comment:
Yup - this is where it gets subjective - FWIW here is the loading gif I use in ECL Watch:  https://github.com/GordonSmith/HPCC-Platform/blob/master/esp/src/eclwatch/img/loadingBar.gif

But I also like the way feedly.com does its progress (at the top of the screen):  http://feedly.com/i/index